src/main/java/com/anaplan/client/AnaplanService.java

```diff
@@ -0,0 +1,228 @@
+package com.anaplan.client;
```

Comment:
is this our code? If so it shouldn't use the com.anaplan prefix, and should have a copyright header.

Reply:
or does it need to be here because required classes like Program are not public?

Reply:
Yeah it's our code. And yes it has to be in this package because the required methods in Program are package-private.

The copyright header is added.